### ISSUE TITLE
Error when entering theorem with doc comment

### ISSUE THREAD

**Issue by vasnesterov** [lean-dojo/LeanDojo#149]
It seems that LeanDojo cannot enter a theorem if there is a documentation comment above it. I found out that this occurs because writing anything between the doc comment and the definition in Lean causes an error. For example, we cannot write
```lean
/--
Some docs
-/
set_option maxHeartbeats 0 in
theorem foo : 1 = 1 := by sorry
```
and after modifying the .lean file [this](https://github.com/lean-dojo/LeanDojo/blob/6710e067dd1f3a19fb07db621a0b38f9f1d722d3/src/lean_dojo/interaction/dojo.py#L417) way Lake is unable to build it.

Minimal example:
```py
from lean_dojo import *

repo = LeanGitRepo(
    "https://github.com/leanprover-community/mathlib4",
    "3c307701fa7e9acbdc0680d7f3b9c9fed9081740",
)

theorem = Theorem(repo, "Mathlib/Topology/Order/Basic.lean", "TFAE_mem_nhdsWithin_Iio")

dojo, state_0 = Dojo(theorem).__enter__()
```
This code raises the exception
```
DojoCrashError: Mathlib/Topology/Order/Basic.lean:926:2: error: [fatal] not_a_theorem
```

**Comment by yangky11** [comment]
Thanks for reporting this bug. I just fixed it in the latest version (`v1.8.0`). Could you please try and let me know?

**Comment by yangky11** [comment]
Closing due to inactivity.
